src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,6 +1,14 @@
 package frc.robot;
 
 public class RobotMap {
+public final static double KP12 = 0.0000018;
+public final static double KF12 = 0.00018;
+public final static double KP14 = 0.0000018;
+public final static double KF14 = 0.00018;
+
```

Comment:
State that those values belong to the Shooter

Reply:
I fixed it.